src/main/java/com/ajou/hertz/domain/administrative_area/controller/AdministrativeAreaControllerV1.java

```diff
@@ -0,0 +1,68 @@
+package com.ajou.hertz.domain.administrative_area.controller;
+
+import static com.ajou.hertz.common.constant.GlobalConstants.*;
+
+import java.util.List;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.ajou.hertz.domain.administrative_area.dto.AdministrativeAreaDto;
+import com.ajou.hertz.domain.administrative_area.dto.response.AdministrativeAreaEmdResponse;
+import com.ajou.hertz.domain.administrative_area.dto.response.AdministrativeAreaSggResponse;
+import com.ajou.hertz.domain.administrative_area.dto.response.AdministrativeAreaSidoResponse;
+import com.ajou.hertz.domain.administrative_area.entity.AdministrativeAreaEmd;
+import com.ajou.hertz.domain.administrative_area.entity.AdministrativeAreaSgg;
+import com.ajou.hertz.domain.administrative_area.entity.AdministrativeAreaSido;
+import com.ajou.hertz.domain.administrative_area.repository.AdministrativeEmdRepository;
+import com.ajou.hertz.domain.administrative_area.repository.AdministrativeSggRepository;
+import com.ajou.hertz.domain.administrative_area.repository.AdministrativeSidoRepository;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+
+@Tag(name = "행정구역 관련 API")
+@RestController
+@RequestMapping("/v1/administrative-areas")
+@RequiredArgsConstructor
+public class AdministrativeAreaControllerV1 {
+	private final AdministrativeSidoRepository administrativeSidoRepository;
+	private final AdministrativeSggRepository administrativeSggRepository;
+	private final AdministrativeEmdRepository administrativeEmdRepository;
+
+	@Operation(summary = "행정구역 시도 조회", description = "행정구역 시도를 조회합니다.")
+	@GetMapping(value = "/sido", headers = API_MINOR_VERSION_HEADER_NAME + "=" + 1)
+	public AdministrativeAreaSidoResponse getSidoList() {
```

Comment:
Method name도 version에 맞춰 적절히 수정해주세요.
```suggestion
	public AdministrativeAreaSidoResponse getSidoListV1_1() {
```